### DISCUSSION TITLE
minio and iSCSI

### DISCUSSION THREAD

**Discussion by cskoszeg** [minio/minio#15370] [Q&A] [answered]
Hi,
Is it possible to use minio with iSCSI for storage over NFS? If yes should I expect performance degradation with the setup?
Would appreciate having an answer from who has experience around.

**Comment by klauspost** [comment, marked as answer]
It may be possible, but it may either be slow or unreliable, or both.

You are of course welcome to test, but it is not a setup we would recommend.

**Reply by cskoszeg** [reply]
Thanks for the quick answer.
